docs/tutorials/dronecan_px4_flight_controller.rst

```diff
@@ -0,0 +1,449 @@
+        Reserved:  81 blocks
+        Allocated: 26 blocks
+
+    UAVCAN node status:
+        Internal failures: 0
+        Transfer errors:   1
+        RX transfers:      1317
+        TX transfers:      3383
+
+    CAN1 status:
+        HW errors: 1
+        IO errors: 1
+        RX frames: 919
+        TX frames: 3800
+    CAN2 status:
+        HW errors: 50147
+        IO errors: 53382
+        RX frames: 0
+        TX frames: 554
+
+    ESC outputs:
+    INFO  [mixer_module] Param prefix: UAVCAN_EC
+    control latency: 0 events, 0us elapsed, 0.00us avg, min 0us max 0us 0.000us rms
+    INFO  [mixer_module] Mixer loaded: no
+    INFO  [mixer_module] Driver instance: 0
+    Channel Configuration:
+    Channel 0: value: 0, failsafe: 0, disarmed: 65535, min: 1, max: 8191
+    Channel 1: value: 0, failsafe: 0, disarmed: 65535, min: 1, max: 8191
+    Channel 2: value: 0, failsafe: 0, disarmed: 65535, min: 1, max: 8191
+    Channel 3: value: 0, failsafe: 0, disarmed: 65535, min: 1, max: 8191
+    Channel 4: value: 0, failsafe: 0, disarmed: 65535, min: 1, max: 8191
+    Channel 5: value: 0, failsafe: 0, disarmed: 65535, min: 1, max: 8191
+    Channel 6: value: 0, failsafe: 0, disarmed: 65535, min: 1, max: 8191
+    Channel 7: value: 0, failsafe: 0, disarmed: 65535, min: 1, max: 8191
+    Servo outputs:
+    INFO  [mixer_module] Param prefix: UAVCAN_SV
+    control latency: 0 events, 0us elapsed, 0.00us avg, min 0us max 0us 0.000us rms
+    INFO  [mixer_module] Mixer loaded: no
+    INFO  [mixer_module] Driver instance: 0
+    Channel Configuration:
+    Channel 0: value: 0, failsafe: 0, disarmed: 0, min: 0, max: 0
+    Channel 1: value: 0, failsafe: 0, disarmed: 0, min: 0, max: 0
+    Channel 2: value: 0, failsafe: 0, disarmed: 0, min: 0, max: 0
+    Channel 3: value: 0, failsafe: 0, disarmed: 0, min: 0, max: 0
+    Channel 4: value: 0, failsafe: 0, disarmed: 0, min: 0, max: 0
+    Channel 5: value: 0, failsafe: 0, disarmed: 0, min: 0, max: 0
+    Channel 6: value: 0, failsafe: 0, disarmed: 0, min: 0, max: 0
+    Channel 7: value: 0, failsafe: 0, disarmed: 0, min: 0, max: 0
+
+    Sensor 'gnss':
+    name: uavcan_gnss
+
+    Sensor 'mag':
+    name: uavcan_mag
+
+    Online nodes (Node ID, Health, Mode):
+        5 OK         OPERAT    
+        127 OK         OPERAT    
+
+    uavcan: cycle time: 49750 events, 2398095us elapsed, 48.20us avg, min 13us max 152183us 730.617us rms
+    uavcan: cycle interval: 49750 events, 2701.02us avg, min 26us max 152824us 1225.585us rms
+
+Test Throttle
+##############
+
+Older PX4 Firmware (v1.13.2 and earlier)
+-----------------------------------------
+It is also possible to test spinning the module in this console using the motor_test command. Typing “motor_test -h” in the console will provide help on how to use the command. 
+As a basic example, to spin the motor with an ESC index of 1 at 20% of its maximum, you could use the command “motor_test test -m 1 -p 20”. The image below shows some 

```

Comment:
rather than putting the code into quotes, how about a code block: .. code-block::